Refit/Attributes.cs

```diff
@@ -206,6 +206,19 @@ public HeaderAttribute(string header)
         public string Header { get; }
     }
 
+    [AttributeUsage(AttributeTargets.Parameter)]
```

Comment:
This type should have doc comments on it and its members

Reply:
I have added some docs to it :)